.storybook/stories.test.js

```diff
@@ -0,0 +1,13 @@
+import initStoryshots from '@storybook/addon-storyshots';
+import { puppeteerTest } from '@storybook/addon-storyshots-puppeteer';
+import path from 'path';
```

Comment:
Have you looked into configuring this in eslint?  Would be a nice-to-have

I had a look at https://eslint.org/docs/rules/sort-imports but that's not what we want at all

https://github.com/lydell/eslint-plugin-simple-import-sort looks interesting but I don't see how it can have the complexity that we want
* differentiating between WordPress, Node, and external dependencies, and then adding headers

Reply:
See https://github.com/WordPress/gutenberg/blob/trunk/packages/eslint-plugin/docs/rules/dependency-group.md 😉